libs/explorers/comparison-tool/src/lib/help-links/help-links.component.html

```diff
@@ -12,3 +12,6 @@
     </div>
   </div>
 </div>
+
+<explorers-legend-panel />
+<explorers-visualization-overview-panel />
```

Comment:
HelpLinksComponent now encapsulates the LegendPanelComponent and VisualizationOverviewPanelComponent